cognite/client/_api/workflows.py

```diff
@@ -396,7 +396,9 @@ def run(
         if metadata is not None:
             body["metadata"] = metadata
 
-        response = self._post(url_path=f"/workflows/{workflow_external_id}/versions/{version}/run", json=body)
+        response = self._post(
+            url_path=f"/workflows/{quote(workflow_external_id, '')}/versions/{quote(version, '')}/run", json=body
```

Comment:
FYI, there exist a utility function for this already, `interpolate_and_url_encode`

Reply:
Yes, the other methods are using the quoting method. I'd prefer to keep the method in the file the same.

Reply:
I leave that up to you 😊